src/multisig/multisig_account_kex_impl.cpp

```diff
@@ -716,11 +724,44 @@ namespace multisig
       m_threshold,
       m_signers,
       expanded_msgs,
-      exclude_pubkeys,
+      get_kex_exclude_pubkeys(),
       result_keys_to_origins_map).get_msg();
 
     // finish account update
     finalize_kex_update(rounds_required, std::move(result_keys_to_origins_map));
   }
+  //-----------------------------------------------------------------
+  // multisig_account: EXTERNAL
+  //-----------------------------------------------------------------
+  multisig_kex_msg multisig_account::multisig_kex_round_booster(const std::uint32_t threshold,
+    const std::uint32_t num_signers,
+    const std::vector<multisig_kex_msg> &expanded_msgs) const
+  {
+    // the messages passed in should be intended to complete the next round
+    const std::uint32_t expected_msgs_round = m_kex_rounds_complete + 1;
+
+    CHECK_AND_ASSERT_THROW_MES(num_signers > 1, "Must be at least one other multisig signer.");
+    CHECK_AND_ASSERT_THROW_MES(num_signers <= config::MULTISIG_MAX_SIGNERS, "Too many multisig signers specified.");
+    CHECK_AND_ASSERT_THROW_MES(expected_msgs_round < multisig_kex_rounds_required(num_signers, threshold),
+      "Tried to boost a round that can't exist (input kex messages probably correspond to the final round).");
+    CHECK_AND_ASSERT_THROW_MES(expanded_msgs.size() > 0, "At least one input kex message expected.");
+
+    // sanitize pubkeys from input msgs
+    std::unordered_map<crypto::public_key_memsafe, std::unordered_set<crypto::public_key>> pubkey_origins_map;
+    std::uint32_t round = multisig_kex_msgs_sanitize_pubkeys(m_base_pubkey,
+      expanded_msgs,
+      get_kex_exclude_pubkeys(),
+      pubkey_origins_map);
+    CHECK_AND_ASSERT_THROW_MES(round == expected_msgs_round, "Kex messages were not for expected round.");
+
+    // dummy variable to satisfy function signature
+    // - do not save the DH derivations produced
+    // - the booster is not used directly for building a multisig account
+    std::unordered_map<crypto::public_key_memsafe, std::unordered_set<crypto::public_key>> pubkey_mapping_dummy;
+
+    // produce a kex message for the next round with given input pubkeys
+    return multisig_kex_make_next_msg(m_base_privkey, round + 1, threshold, num_signers,
```

Comment:
I can't find the implementation of this function. Where is it? The builds supposedly pass, but it doesn't appear in the current codebase either. What am I missing?